collector/lib/CollectorConfig.cpp

```diff
@@ -102,48 +121,60 @@ void CollectorConfig::InitCollectorConfig(CollectorArgs* args) {
   if (args) {
```

Comment:
Not something for this PR necessarily, but this init function is huuuuuuge. I wonder if we could break it up a little? wdyt?

Reply:
Yeah, it's gotten way out of hand. We can probably figure some way to split sections of the configuration up and just have private/protected methods that handle those individually or something. I might give it a shot in a follow up.